e2etest/container/config.go

```diff
@@ -5,19 +5,24 @@ package container
 type ImageConfig struct {
 	BitcoindRepository string
 	BitcoindVersion    string
+	BabylonRepository  string
+	BabylonVersion     string
 }
 
 //nolint:deadcode
 const (
 	dockerBitcoindRepository = "lncm/bitcoind"
 	dockerBitcoindVersionTag = "v27.0"
+	dockerBabylondRepository = "babylonlabs/babylond"
+	dockerBabylondVersionTag = "8e0222804ed19b18d74d599b80baa18f05e87d8a" // this is built from commit b1e255a
```

Comment:
so this should be manually updated to be consistent with the version in `go.mod`, correct? How to get the docker version?

Reply:
It should be a tag, like v0.9.3, but our CI wasn't setup at the time to build for arm64. So yes, it should reflect go mod version